docs/docs/prototype-an-assistant.mdx

```diff
@@ -4,7 +4,6 @@ sidebar_label: 'Prototype an Assistant'
 title: 'Prototype an Assistant'
 ---
 import useBaseUrl from '@docusaurus/useBaseUrl';
-import Prototyper, { Chat, DownloadButton, TrainButton } from '@site/src/components/prototyper';
```

Comment:
I don't think it's a good idea to rely on global imports instead of importing exactly what we need in each files. I would agree that maybe for `<Tabs>` there is a benefit (it's standard, it's from docusaurus, it's used in a lot of places), but for the case of the Prototyper, and surely other components, I think it's a bad idea and doesn't really scale.

Reply:
I agree the Prototyper is probably too large to import on every page, yes, but importing the common UI elements like Tabs, Buttons etc. is a fairly standard MDX pattern. Will adjust